File: src/AccelByte.Extend.Vivox.Authentication.Server/Vivox/TokenData.cs
// Copyright (c) 2024 AccelByte Inc. All Rights Reserved.
// This is licensed software from AccelByte Inc, for limitations
// and restrictions contact your company contract manager.

using System;

namespace AccelByte.Extend.Vivox.Authentication.Server.Vivox
{
    public class TokenData
    {
        public string Value { get; }

        public TokenClaims Claims { get; }

        internal TokenData(string value, TokenClaims claims)
        {
            Value = value;
            Claims = claims;
        }
    }
}

File: src/main/java/net/accelbyte/extend/serviceextension/vivox/TokenData.java
package net.accelbyte.extend.serviceextension.vivox;

import lombok.Data;

@Data
public class TokenData {
    public String value;
    public TokenClaims claims;
    TokenData(String value, TokenClaims claims) {
        this.value = value;
        this.claims = claims;
    }
}
